### ISSUE TITLE
webdriver.executeScript wait a long time(26s) before page load

### ISSUE THREAD

**Issue by zixia** [Chatie/wechaty#2]
It seems that webdriver always wait browser window to be stable before run `executeScript`.

the condition of stable is set to be like this:

``` javascript
window.document.readyState === 'complete'
```

But...

If we have a ajax call / http long pulling / etc, the `readyState` will be `interactive` for a long time.
## Solution(?)

make webdrive wait only for readyState is `loading`
## See Also
1. How to `executeScript` before page load by WebDriver in selenium? https://stackoverflow.com/questions/37071807/how-to-executescript-before-page-load-by-webdriver-in-selenium
2. A ghost driver on phantomjs quick & dirty patch https://github.com/detro/ghostdriver/issues/334#issuecomment-41361564
3. Attempt to make selenium tests more reliable https://github.com/18F/calc/pull/267
4. [PhantomJS 1.9 has introduced `resourceTimeout`](http://stackoverflow.com/a/18837957/1123955)
## PhantomJS Log

When waiting, it output lots of logs like this:

```
[DEBUG - 2016-08-07T10:21:57.754Z] Session [b69451a0-5c88-11e6-bed4-fb6eca6b39c0] - _execFuncAndWaitForLoadDecorator - Page Loading in Session: true
```
## PhantomJS Code

Do not know what SIGNAL is, what SLOT is...

https://github.com/ariya/phantomjs/blob/096d0da87ee6d94f3eed84446c0eb5176566d54a/src/webpage.cpp#L396
## GhostDriver Code

https://github.com/detro/ghostdriver/blob/f976007a431e634a3ca981eea743a2686ebed38e/src/session.js#L233

maybe change `_isLoading()` to this?

``` javascript
    /**
     * Is any window in this Session Loading?
     * @returns "true" if at least 1 window is loading.
     */
    _isLoading = function() {
        var wHandle;

        for (wHandle in _windows) {
//            if (_windows[wHandle].loading) {
            if (_windows[wHandle].document.readyState) === 'loading')
                return true;
            }
        }

        // If we arrived here, means that no window is loading
        return false;
    }
```

**Comment by bastrich** [comment]
@zixia Use the following option
capabilities.setCapability(CapabilityType.PAGE_LOAD_STRATEGY, "none);

**Comment by zixia** [comment]
@bastrich Thanks for the information!

However, I can not found `setCapability` method in selenium-webdriver nodejs module.

There's nothing outputed by the following commands:

```shell
01:38:19 (master) ~/git/wechaty/node_modules/selenium-webdriver$ grep setCapability * -R
01:38:55 (master) ~/git/wechaty/node_modules/selenium-webdriver$ grep CapabilityType * -R
```

After a search on selenium repository at here: https://github.com/SeleniumHQ/selenium/search?utf8=%E2%9C%93&q=PAGE_LOAD_STRATEGY , I also did not found any more information about it. 

Could you confirm it works with nodejs and chrome? If so, can you post a runnable nodejs code for it?

Thanks!

**Comment by zixia** [comment]
This issue should be fixed at v0.10 or above.
